### PR TITLE
Topic related views

### PR DESCRIPTION
**Pull request by CHenrie**

### CODE REVIEW

**Comment by CHenrie** [comment]
@jbest84  Please Review

File: src/Format.js [hunk 1]
```diff
@@ -252,6 +252,51 @@ define('Mobile/SalesLogix/Format', [
...
+         * `Lee Hogan` -> `LH`
+         * @param val
+         * @returns {String}
+         */
+        formatUserInitial: function(user) {
+            var firstLast = this.resolveFirstLast(user),
+                initials = [firstLast[0].substr(0, 1)];
+
+            if (firstLast[1])
+                initials.push(firstLast[1].substr(0, 1));
```
Comment (jbest84) [inline src/Format.js:267]: Formatting, statement should have brackets.

File: src/Format.js [hunk 2]
```diff
@@ -252,6 +252,51 @@ define('Mobile/SalesLogix/Format', [
...
+       * `Hogan, Lee` -> `Lee Hogan`
+       * @param val
+       * @returns {String}
+       */
+        resolveFirstLast: function(name) {
+            var firstLast = [];
+            if (name.indexOf(' ') !== -1) {
+                var names = name.split(' ');
+                if (names[0].indexOf(',') !== -1)
+                    firstLast = [names[1], names[0].slice(0, -1)];
```
Comment (jbest84) [inline src/Format.js:293]: Formatting, statement should have brackets.

File: src/Format.js [hunk 3]
```diff
@@ -252,6 +252,51 @@ define('Mobile/SalesLogix/Format', [
...
+        /**
+       * Takes a string input and the user name to First amd Last name 
+       * `Hogan, Lee` -> `Lee Hogan`
+       * @param val
+       * @returns {String}
+       */
+        resolveFirstLast: function(name) {
+            var firstLast = [];
+            if (name.indexOf(' ') !== -1) {
+                var names = name.split(' ');
```
Comment (jbest84) [inline src/Format.js:291]: Multiple vars

**Pull request merged**